### ISSUE TITLE
QUESTION: is possible to print output by stdout instead of using less ?

### ISSUE THREAD

**Issue by asdf8601** [profclems/glab#656]
**Describe the feature or problem you'd like to solve**

I'm wondering if there is a way to print out the output command by stdout instead of using the pager. For example I would like to print by stdout the output of `glab issue ls -P 4` or `glab issue ls --search key` so I can work with this information in other commands without keeping `less` open.

Currently, I'm using `glab issue ls -P 4 | tr "" ""` as workaround. Maybe you know a better way to archieve the same behaviour.

Thanks in advance!

**Comment by profclems** [comment]
Yes it is possible. `glab` uses the `PAGER` env variable so you can override it:
```
export PAGER=""
glab issue ls -P 4
```
or
```
PAGER= glab issue ls -P 4
```
or you can set the `PAGER` to `cat`.

We'd have to fix this issue #320 so that users can use `GLAB_PAGER` instead of unsetting the general `PAGER` variable

**Comment by asdf8601** [comment]
Thank you @profclems for your quick answer! 

> We'd have to fix this issue #320 so that users can use GLAB_PAGER instead of unsetting the general PAGER variable

Awesome, That's looks very nice, as usual! 

> or you can set the PAGER to cat.

Until #320 will be merged, I will use a bash alias using `cat` as `PAGER`: `clab="PAGER=cat glab"`.

All done to me, thank you!
